doc/howto/manage_bsnl_sms_reminders.md

```diff
@@ -0,0 +1,45 @@
+## Sending reminder SMS via BSNL
+
+We use BSNL to send reminder messages in India. This requires us to comply with DLT regulations, for which we need to maintain
+a mapping between IDs assigned by DLT and our messages (identified by locale keys).
+
+This is a how-to on interacting with the BSNL and DLT dashboards and maintaining the required mapping with `simple-server`.
+
+### Managing JWT tokens
+<TODO>
+
+### Adding new notification strings
+- Add the new notification strings to the locale file (in `config/notifications/locale`)
+- Upload the notification template to the DLT dashboard. The name should follow the format: `locale.locale_key`. Some examples:
+  - `en.notifications.set01.basic`
+  - `hi-IN.notifications.set03.basic`
+  - `ti-ET.notifications.set03.basic_repeated.first`
+
+  The locale name needs to be hyphenated (`hi-IN` and not `hi_IN`). This format is used to associate the templates on BSNL to locale keys so it's an important step.
+- The template should show up on the BSNL dashboard in a few hours. You will need to name the variables
+in the template.
+
+<TODO flesh out these notes on dashboard usage and end-to-end flow, maybe add a video>
+
+### Maintaining template IDs
+After the new template has been added to the BSNL dashboard, we need to pull in it's `DLT Template ID` before it can be used as a reminder SMS. We have a script to help with it.
+
+**Note:** You will need production credentials to run the script.
```

Comment:
Add a note about not using token 1 anywhere other than production. Or whichever token we want to reserve for prod use.

Reply:
I've instructed to copy the JWT token from production so they don't have to generate a new one. Should be ok?

Reply:
Hmm, I guess that works for now. Let's come back and think about developer UX later though. Should be easy enough to set up our scripts to use a different token id for dev use.